docs/articles/New-Custom-Element.md

```diff
@@ -0,0 +1,297 @@
+# How to create new custom element

```

Comment:
Why this page is on GitHub? 

This place should serve as a GitHub resources.

Reply:
This should be still under learn portal - if we want to migrate thees resources, we should do it globally.

Reply:
Let's delete this page => by udpating the react remplate readme - we have a substitute for that: https://github.com/kontent-ai/custom-element-template-react/pull/2/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5